.github/workflows/test-create-gcloud-instance.yml

```diff
@@ -0,0 +1,20 @@
+name: Test create gcloud install
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Add something like https://github.com/Homebrew/actions/blob/560e336611c852f0ab459a16e3041001fc39ec7b/.github/workflows/cancel-previous-runs.yml#L5-L9 so it's tested in PRs like this.